### DISCUSSION TITLE
stash built list

### DISCUSSION THREAD

**Discussion by gtoilet** [runelite/runelite#15494] [Ideas]
would be nice to know which stashes i need built before i get that clue
any way to see which ones are already built thru RL?

**Comment by Nightfirecat** [comment]
There's no existing plugin I know of which would show you all your stash units. You can go to Watson's house to see an ingame UI which shows the units you have and haven't built.

**Comment by gtoilet** [comment]
well when i get some clues RL knows i got that stash built
so theres got to be a list somewhere in the RL unless it pulls each clue to see if you got that stash at that time

**Reply by Nightfirecat** [reply]
There is a script which can run to check whether a stash unit is built that can be run from anywhere ingame, however the client does not have the knowledge of whether said stash unit is filled, so it's not really useful for anything more than we already use it for.
